recipes/MultiWOZ/response_generation/hparams/train_gpt.yaml

```diff
@@ -88,7 +88,7 @@ test_dataloader_options:
 padding_mask: !name:speechbrain.lobes.models.transformer.Transformer.get_key_padding_mask
 
 # gpt model
-gpt_model: !new:speechbrain.lobes.models.huggingface_gpt.HuggingFaceGPT
+gpt_model: !new:speechbrain.lobes.models.huggingface_transformers.gpt.GPT
```

Comment:
@poonehmousavi : this got changed because we created a dir for all HF models (huggingface_transformers), in which gpt.py is located. Long story but we omitted "huggingface" everywhere in the py file names and also the name of the model classes because we believe that the huggingface_transformers dir implicitly indicates the HF nature of these models.